spec/support/.rubocop.yml

```diff
@@ -0,0 +1,2 @@
+inherit_from:
+  - ../../.rubocop.yml
```

Comment:
LOL - so we have a main rule, which we're overriding, then overriding the override back to the main rule? 😆

I get the feeling we're really pushing against what rubocop wants us to do, but since it's not affecting any app or spec code I'm going to say it's fine. 

I feel like this sort of thing really does need a good commit message explaining it though, because otherwise someone will look at that file in a few months and won't have any idea why it's there.